lib/Db/OldPasswordMapper.php

```diff
@@ -47,7 +47,7 @@ public function getOldPasswords($uid, $length, $excludeForceExpired = true) {
 			->orderBy('id', 'desc')
 			->setMaxResults($length);
 		if ($excludeForceExpired) {
-			$qb->andWhere($qb->expr()->neq('password', OldPassword::EXPIRED));
+			$qb->andWhere($qb->expr()->neq('password', $qb->expr()->literal(OldPassword::EXPIRED)));
```

Comment:
ok, so you were right ...

Reply:
yep, this blew up during unit testing...